setup.py

```diff
@@ -22,29 +22,6 @@
 
 here = os.path.abspath(os.path.dirname(__file__))
 
-def parse_reqs(req_path='./requirements.txt'):
```

Comment:
I'm installing the requirements and the app itself in 2 steps, so that when building the docker image the requirements will be cached, otherwise it takes forever to install packages like cyvcf2

Reply:
Again suggesting to use poetry for this. In the container you can copy the `poetry.lock` file and install that with [micropipenv](https://towardsdatascience.com/a-bridge-to-two-python-dependency-pinning-worlds-micropipenv-5da674f38e89)

Reply:
Another PR!